### DISCUSSION TITLE
Noise Reduction Model for every Sound

### DISCUSSION THREAD

**Discussion by Nexeration** [Anjok07/ultimatevocalremovergui#696] [Ideas]
There are quite a few options for vocal noise reduction now which is great.
A lot of the times musicians are also facing the problem of unwanted noise in other recording like for guitars/drums/field recordings. 

If Ultimate Vocal Remover would get a model which concentrates on removing noise in every sound by training the model on noise sounds in general it would be the first effective tool for this case. Sure there is RX by Izotope but it's not that good.

**Comment by Excurl** [comment]
Pls get this done..
